controllers/flowlogspipeline/flp_common_objects.go

```diff
@@ -45,6 +45,7 @@ const (
 	conntrackTerminatingTimeout = 5 * time.Second
 	conntrackEndTimeout         = 10 * time.Second
 	conntrackHeartbeatInterval  = 30 * time.Second
+	clusterNameDefault          = "defaultCluster"
```

Comment:
```suggestion
```

Reply:
Changed default to blank, but in that case we take the clusterName from Openshift.